wolfCLU/README.md

```diff
@@ -1,12 +1,12 @@
-#wolfCLU
+# wolfCLU
 
-This is the wolfSSL:  Command Line Utility (wolfCLU).
+This is the wolfSSL: Command Line Utility (wolfCLU).
 
-##wolfSSL Install
+## wolfSSL Install
 
 To use this feature, please configure and install wolfssl with the following commands:
 
-    ./configure --enable-pwdbased --enable-opensslextra && make && make check
+    ./configure --enable-pwdbased --enable-opensslextra --enable-base64encode && make && make check
```

Comment:
This check in wolfssl/wolfcrypt/coding.h should eliminate the need to configure with enable-base64encode since we already say to configure with opensslextra.

```
 37 #if defined(OPENSSL_EXTRA) || defined(SESSION_CERTS) || defined(WOLFSSL_KEY_GEN) \
 38    || defined(WOLFSSL_CERT_GEN) || defined(HAVE_WEBSERVER) || !defined(NO_DSA)  
 39     #ifndef WOLFSSL_BASE64_ENCODE                                               
 40         #define WOLFSSL_BASE64_ENCODE                                           
 41     #endif                                                                      
 42 #endif
```

Otherwise looks good! Thanks David!